dataland-frontend/src/components/general/CompanyDataTable.vue

```diff
@@ -130,6 +130,7 @@ export default defineComponent({
         props: {
           header: modalTitle,
           modal: true,
+          dismissableMask: true,
```

Comment:
As discussed: 
Please also test this. e.g.: In "DataIntegrity.ts" - Line 155: Instead of clicking the close-icon, click to the background.

Reply:
done.

Reply:
ok